packages/pageheader/README.md

```diff
@@ -0,0 +1,24 @@
+# PageHeader Name
```

Comment:
What, if any, relationship does this have with the AppChrome package? Is this component reusable by two apps? Is there a strategy to migrate DCOS UI to use this?

Reply:
The App chrome is the surrounding frame. Where as this Component is just the heading. Which is containing the Breadcrumb and the actions.

![image](https://user-images.githubusercontent.com/156010/58573826-d682f280-823e-11e9-8852-c1b5ec63ca5c.png)

This is the PageHeader in DC/OS ui

Reply:
The basis of my question was to understand the strategy of having page header in ui-kit which appears to be app-specific. At this point, we have dcos-ui-like app kit components such as AppChrome in ui-kit and I'm wondering if this gets us further in that direction... Or if it is completely dedicated to Dumbledore

Reply:
The PageHeader and all other components should be generic to all applications, although they should be themable to differentiate between different apps.